dartagnan/src/main/java/com/dat3m/dartagnan/wmm/Relation.java

```diff
@@ -70,7 +75,4 @@ public String toString() {
         return names + " := " + getDefinition().getTerm();
     }
```

Comment:
I guess you meant to refer to `isInternal`? This code might have become obsolete now. IIRC, it's only purpose was to let `Refinement` know which relations it should ignore, because (as you said) those involve non-visible events.